Microsoft.O365.Security.Native.ETW/EventTraceProperties.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include <krabs.hpp>
+
+namespace Microsoft {
```

Comment:
nit: could you inline these namespace declarations and reduce the level of indent below, following the pattern of files like [EventRecord.hpp](https://github.com/microsoft/krabsetw/blob/master/Microsoft.O365.Security.Native.ETW/EventRecord.hpp)?

Ex: `namespace Microsoft { namespace O365 { namespace Security { namespace ETW {`

Reply:
*doh* I fixed this a couple of times, but Visual Studio's auto-format kept changing it back.